### PR TITLE
Fixed: Make customizations valid

### PR DESCRIPTION
**Pull request by ahankinson**
This PR follows on from #1171 and fixes errors in the customizations.

It should be merged after #1171, since it includes the changes from that PR.

The commits in this PR are separated by the individual changes and the commit message contains a short description on what the problem was.

### CODE REVIEW

File: customizations/mei-basic.xml [hunk 1]
```diff
@@ -11,6 +12,10 @@
                     <resp>Authored by</resp>
                     <name xml:id="JK">Johannes Kepper</name>
                 </respStmt>
+                <respStmt>
+                    <resp>Revisions</resp>
+                    <name xml:id="LP">Laurent Pugin</name>
+                </respStmt>
```
Comment (kepper) [inline customizations/mei-basic.xml:18]: Didn't you just revise it as well? Good catch, but we should be more generous / correct when giving credit…

**Review by kepper — DISMISSED** [review]
In general, all fine. I would suggest to also give credit to @ahankinson, though…

File: customizations/mei-basic.xml (hunk 1 above)
Comment (ahankinson) [reply customizations/mei-basic.xml:18]: No, I just fixed the bugs. :-D

Comment (ahankinson) [reply customizations/mei-basic.xml:18]: The credit is also to "The MEI Community" so I'm covered by that.

**Review by bwbohl — COMMENTED** [review]
Still got one validation error in mi-CMN.xml, at least on my machine.
System-ID: /Volumes/VCS/GitHub/bwbohl/music-encoding/customizations/mei-CMN.xml
Haupt-Validierungsdatei: /Volumes/VCS/GitHub/bwbohl/music-encoding/customizations/mei-CMN.xml
Schema: /Volumes/VCS/GitHub/bwbohl/music-encoding/source/validation/mei-customizations.sch
Programmname: ISO Schematron
Fehlerlevel: error
Beschreibung: There is no model class with name "model.scorePart.mensuralAndNeumes".
Anfang: 107:26

**Comment by ahankinson** [comment]
@bwbohl fixed.

**Review by bwbohl — APPROVED** [review]

**Comment by bwbohl** [comment]
thx @ahankinson

**Review by rettinghaus — APPROVED** [review]

**Comment by bwbohl** [comment]
thx @rettinghaus!

**Pull request merged**